docs/README.md

```diff
@@ -21,65 +21,72 @@ Be the customer, not the product. We make money by selling a note-taking app, no
 
 | Payment Option | Monthly Fee    |
 |----------------|----------------|
-| Monthly        | $2.99 per 30gb |
+| Monthly        | $2.99 per 30GB |
 
 ### Developer Friendly
-We also provide a CLI tool that will fit right into your favorite chain of piped-together unix commands. Search your notes with `fzf`, edit them with `vim`, and schedule backups with `cron`. When scripting doesn't cut it, use our Rust library for a robust programmatic interface with Lockbook.
+We also provide a CLI tool that will fit right into your favorite chain of piped-together Unix commands. Search your notes with `fzf`, edit them with `vim`, and schedule backups with `cron`. When scripting doesn't cut it, use our Rust library for a robust programmatic interface with Lockbook.
+
+## How To Install
```

Comment:
I put this in the readme to onboard new users quickly (and because the install instructions are simple enough that they don't need their own guides anymore)